### DISCUSSION TITLE
video from other websites

### DISCUSSION THREAD

**Discussion by Runar-A** [sampotts/plyr#2673] [Q&A]
I am currently creating a personal website where I can watch videos, until now I have been downloading videos and using the html5 video player. I noticed that my storage was getting low and wondered if I could do something like the Youtube player but for different websites.

let's take diode.zone as an example, it's very similar to Youtube and the embed links look pretty similar, but when I try to add the embed links to my html using <iframe>, the video comes out without the Plyr customization/raw

Does anyone have an answer?
sorry for bad english

**Comment by belbernard** [comment]
As far as I understand, all you need to specify is the URL of the video file.

Let's assume your site is programmed in WordPress. Just try to create a HTML block with the following code and you'll see a video of my own site on your site:

<video id="bb-video" controls style="width: 100%;">
  <source src="https://lebonheurestpossible.org/cloud/AseemMalhotra-JoeRogan.mp4" type="video/mp4">
  <track kind="captions" src="https://lebonheurestpossible.org/wp-content/uploads/AseemMalhotra-JoeRogan-en-fr.vtt" srclang="fr" label="&#x1f1eb;&#x1f1f7; - &#x1f1ec;&#x1f1e7;&nbsp;French-English" default>
  <track kind="captions" src="https://lebonheurestpossible.org/wp-content/uploads/AseemMalhotra-JoeRogan-en.vtt" srclang="en" label="&#x1f1ec;&#x1f1e7;&nbsp;English">
</video>

<script src="https://cdn.plyr.io/3.6.8/plyr.js"></script>
<link href="https://cdn.plyr.io/3.6.8/plyr.css" rel="stylesheet" />

<script>
  document.addEventListener('DOMContentLoaded', () => {
    var player = new Plyr('bb-video', {
      controls: ['play', 'progress', 'volume', 'captions', 'settings', 'fullscreen'],
      captions: { active: true }
    });
</script>

Here you see that the video URL is https://lebonheurestpossible.org/cloud/AseemMalhotra-JoeRogan.mp4"
However, it is not hosted on my site. I have set up a redirection (via htaccess) to access the MP4 file in my Google Cloud storage.
